models/tasks.js

```diff
@@ -154,15 +165,29 @@ const fetchUserTasks = async (username, statuses = []) => {
 
     const promises = tasks.map(async (task) => fromFirestoreData(task));
     const updatedTasks = await Promise.all(promises);
-    return updatedTasks;
+    const taskList = updatedTasks.map((task) => {
+      task.status = TASK_STATUS[task.status.toUpperCase()];
```

Comment:
what if that task.status is not present in the `TASK_STATUS` ?

Reply:
We are showing the value which is already present as status(will make the change in next commit). Like here,https://github.com/Real-Dev-Squad/website-backend/blob/ccc712575ba3dfb7e35a0f9fb92659d2fcbccce1/models/tasks.js#L119